src/mosaic-node-manager.cc

```diff
@@ -99,9 +98,36 @@ namespace ns3 {
             BuildingsHelper::Install (eNodeB);
             BuildingsHelper::MakeMobilityModelConsistent();  
 
-            m_groupL2Address = 0x00;
+            m_groupL2Address = 0x01;
```

Comment:
Still unclear what this change in constant means.

Reply:
I had been passing this value to another class and implemented a condition to ensure successful transmission by checking the boolean `!m_groupL2Address`. However, it has become unnecessary to pass it to that class, and I forgot to revert the change.